Automattic/pocket-casts-ios
# [Up Next Shuffle] Enable FF

| 📘 Part of: [Up Next Shuffle](https://github.com/orgs/Automattic/projects/1179/views/1)
|:---:|

Fixes #2330

Enable the FF by default. It also updates the CHANGELOG/

## To test

- CI must be 🟢 
- Delete the app and reinstall it
- Check the FF is on by default
- Check the shuffle button is present in the Up Next view

## Checklist
